federated_aws_cli/cache.py

```diff
@@ -167,6 +167,45 @@ def write_id_token(issuer, client_id, token):
         logger.debug("Unable to write id token to: {}".format(path))
 
 
+@_requires_safe_cache_dir
+def read_sts_credentials(role_arn):
+    # Create a sha256 of the endpoint url, so fix length and remove weird chars
```

Comment:
copy paste mistake I imagine
```suggestion
    # Create a sha256 of the IAM role ARN, so fix length and remove weird chars
```